### ISSUE TITLE
New fontello design

### ISSUE THREAD

**Issue by puzrin** [fontello/fontello#484]
New design by @99fella.

---

Icons select:

![select-icons1](https://cloud.githubusercontent.com/assets/319465/13839079/f6294a4a-ec31-11e5-9ef9-fe4e2dcc30eb.png)

Preview / Customize names:

![customize-names](https://cloud.githubusercontent.com/assets/319465/13839228/d7e9c9c8-ec32-11e5-8c66-52205a3eec47.png)

Customize codes (+ ligatures):

![customize-codes](https://cloud.githubusercontent.com/assets/319465/13839260/195d2e68-ec33-11e5-8db3-93e247bab1ce.png)

![customize-codes 1](https://cloud.githubusercontent.com/assets/319465/13839243/f606887e-ec32-11e5-9493-739296e68973.png)

Collections (library):

![select-icons](https://cloud.githubusercontent.com/assets/319465/13839300/69bbe692-ec33-11e5-813b-4d814d24d341.png)

Demo template:

![fontello-dp1](https://cloud.githubusercontent.com/assets/319465/13839305/7401acb8-ec33-11e5-8f76-352de49d2469.png)

**Comment by goodinmech** [comment]
What about a fluid layout for a demo page?

**Comment by gideoncresswell** [comment]
Will you be updating the Instagram logo shortly? https://www.instagram-brand.com/

**Comment by puzrin** [comment]
Published all sources in separate repo https://github.com/fontello/fontello-design.

**Comment by paul2moro** [comment]
Also, the ability to make the icons bigger/smaller would be a plus. 

Also, allowing for multiple config files to be uploaded, to be able to work on multiple font sets without mixing them up.

**Comment by paul2moro** [comment]
also I don't think the new blue theme brings too much in terms of UX. I would've kept it light grey / dark grey + colour where it actually matters (on the buttons).

**Comment by taophp** [comment]
The ligature stuff is really cool for accessibility. Did you planned to go on line with this version soon ?

**Comment by JohnEcks** [comment]
Not sure if this is possible, but I would love the ability to **reposition uploaded svgs** that don't center properly. This happens sometimes when uploading large svgs with non-square content, even if the artboard is square. It would also be useful to **reposition any icons**, as sometimes they may be off by a pixel or two compared to others and need tweaks in CSS compared to the rest in our set.

Would also love the ability to **scale/resize icons and/or uploaded svg** for consistency—individually, for selection, or globally for anything in our set.

Could extend this further from manual tweaks to a **"unify" command** for size, position or both. Sometimes resizing/repositioning of specific icons with CSS is needed to maintain consistency using `font-size` or `vertical-align`, so it would be great to do this while setting up the font so we have less code to bother with later.

It would be nice to **preview your icon set** in multiple ways too, such as if used alone, small and large scales, with `box-shadow`, etc,  or preview with specific fonts using `:before` and `:after` and applying some CSS rules for `color`, `font-size`, maybe even `transform` and such. Could pull fonts in from Google to see how things line up for headings, buttons, anchors, and such.

The more I think of it, this sounds like a new feature set/section. Would probably be time-consuming to make, but I use fontello a lot and this would speed up my workflow, so figured I'd share my ideas.

thanks

**Comment by gangoolie** [comment]
Hi,

Thanks for getting back to me. 

I'll confess: I didn't understand half of the terms in your message!

Good luck with the project. 

Jordan Rolph

**Comment by JohnEcks** [comment]
:)

I'll try and mock something up when I have time to illustrate what I meant.

**Comment by zilly** [comment]
Looking good. How about a bigger searchbox in the upper left? 

Its easy to miss a small box with a fast mouse/cursor.

**Comment by Gotterbild** [comment]
I agree with @roackarul that coloring of icons/headings is bad idea. Would be better to keep colors grayscale.

Another idea I have is that FONTELLO logo in upper right corner should look different. Can make a logo if you're ok with changing it.

Also, I can help with laying the design out in HTML+CSS. I'm good in that. But can't promise to make it fast as I'm pretty busy right now.

**Comment by Jabark** [comment]
Any update on this as we're approaching this threads 4th birthday? Could do with the ability to use ligatures.
